.ci/magician/github/membership.go

```diff
@@ -177,6 +91,10 @@ func IsCoreContributor(user string) bool {
 }
 
 func IsCoreReviewer(user string) bool {
+	// TESTING PURPOSES
```

Comment:
Not sure why this didn't work to make the bot auto-assign a reviewer for this PR.